### DISCUSSION TITLE
Update UI/Canvas to make your bug screenshots easier to work with

### DISCUSSION THREAD

**Discussion by Bablakeluke** [socialstack-core/modules#5] [Announcements]
TLDR: `socialstack i UI/Canvas` for the update itself.

When a component crashes, it displays a "this component has crashed" message on the screen, and indicates the error is in the console. The general idea here was to keep the UI tidy and put the technical reason somewhere that a developer (or piece of error tracking code) can deal with it. However, it was very common for people to send a screenshot of the generic crash prompt which only tells developers which component and not why - leaving the reason as a bit of a mystery. So, this update should mean people who are actively sending screenshots of these component crashes can also screenshot the reason why to greatly aid in the underlying reason.

On a related note, Socialstack will be moving towards making errors more visible on all projects rather than needing to specifically install e.g. Linux tools on prodution clusters.
